src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -106,7 +106,7 @@ export class EditDataGridPanel extends GridParentComponent {
 		this.nativeElement.className = 'editDataGridPanel';
 		this.nativeElement.classList.add('slickgridContainer');
 		this.dataService = dataService;
-		this.actionProvider = this.instantiationService.createInstance(EditDataGridActionProvider, this.dataService, this.onGridSelectAll(), this.onDeleteRow(), this.onRevertRow());
+		this.actionProvider = this.instantiationService.createInstance(EditDataGridActionProvider, this.dataService, this.configurationService, this.onGridSelectAll(), this.onDeleteRow(), this.onRevertRow());
```

Comment:
You shouldn't need to pass in the configuration service anymore - just move the param to the end of the list and createInstance will inject it for you.

Reply:
(once you update it to also use DI)